### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by Maxkoz777**

### CODE REVIEW

File: app_python/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Build and Run
+
+Firstly, you have to clone current project via command:  
+`$ git clone https://github.com/Maxkoz777/devops.git`  
+Secondly, you have to change working directory by command:  
+`$ cd app_python`  
+After that you should install [Flask](https://phoenixnap.com/kb/install-flask) and [Gunicorn](https://flask.palletsprojects.com/en/2.2.x/deploying/gunicorn/)  
+Then you have to execute command in the project root in terminal:  
+`$ gunicorn -w 4 'app:app'`
```
Comment (AKurmazov) [inline app_python/README.md:17]: Nit: It is a good practice to lock your dependencies in a **requirements** file, that would prevent possible users from installing incompatible versions of libraries

**Comment by AKurmazov** [comment]
LGTM. Well done 👍

**Comment by none-word** [comment]
Good job!

**Comment by afonyaa** [comment]
Good job, Great architecture, and following best Java practices is amazing!

**Comment by Homa3030** [comment]
It is nice to see that you used tests and interfaces

**Pull request closed without merging**